uint32 seq
time stamp
string name
novelti/IntMap map
geometry_msgs/Pose init_pose
---
